script.js

```diff
@@ -0,0 +1,44 @@
+const barcontainer = document.querySelector(".baricon");
+const toggle_button = document.querySelector("#nav-toggler");
+function barstogglers() {
+  barcontainer.classList.toggle("change");
+}
+
+function toggleNavMenu() {
+  const menu = document.querySelector(".nav-items");
+  barcontainer.addEventListener("click", barstogglers);
+  if (menu.style.display === "none") {
+    menu.style.display = "block";
+  } else if (menu.style.display == "flex") {
+    menu.style.display = "none";
+  } else {
+    menu.style.display = "none";
+  }
+}
+
+function addNavMenuToggleListener() {
+  const toggleButton = document.querySelector("#nav-toggler");
+  toggleButton.addEventListener("click", toggleNavMenu);
+}
+
+function removeNavMenuToggleListener() {
+  const toggleButton = document.querySelector("#nav-toggler");
+  toggleButton.removeEventListener("click", toggleNavMenu);
+  console.log("from remove navmenu toggle listener");
+}
+
+function checkScreenSize() {
+  const toggleButton = document.querySelector("#nav-toggler");
+  if (window.innerWidth <= 768) {
+    toggleButton.addEventListener("click", addNavMenuToggleListener);
+  } else {
+    toggleButton.removeEventListener("click", removeNavMenuToggleListener);
+    const navMenu = document.querySelector(".nav-items");
+    if (navMenu.classList.contains("active")) {
+      navMenu.classList.remove("active");
+    }
+  }
+}
+
+window.addEventListener("load", checkScreenSize);
+window.addEventListener("resize", checkScreenSize);
```

Comment:
- [ ] Just as the previous reviewer requested, please make sure, When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

https://user-images.githubusercontent.com/69638013/232851944-6298aa1d-44eb-4ae4-8e6a-7b6814b43b35.mp4